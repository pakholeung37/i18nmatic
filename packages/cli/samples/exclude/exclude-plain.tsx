
const ERROR_MESSAGE = "请稍后再试"
const review = {
  id: 1,
  name: "测试",
}
const reviews = [
  {
    id: 1,
    name: "张先生",
  },
  {
    id: 2,
    name: "李女士",
  },
]

const typeNameMap: Record<string, string> = {
  Folder: "文件夹",
  FolderExpanded: "文件夹",
  Model: "持久模型",
  ModelView: "普通模型",
  Statistic: "统计模型",
  Event: "业务事件",
  Action: "扩展服务",
  ServiceConfig: "业务配置",
  Scene: "页面场景",
  ServiceDefinition: "编排服务",
  Flow: "业务流",
  Trigger: "触发器监听",
  EventDefinition: "触发器定义",
  Process: "业务流",
  OverviewProcess: "业务流程",
  WorkflowGroup: "审批流场景",
  NoticeScene: "通知场景",
  PrintScene: "打印场景",
  View: "视图",
  MenuTree: "菜单树",
  TakeCode: "取号中心",
  Connector: "连接器",
  Menu: "菜单",
  Module: "模块",
  StateConfig: "状态配置",
  Validation: "校验规则",
  ImportExportTemplate: "导入导出场景",
  Permission: "功能权限项",
  DataCondition: "数据权限规则",
  DataControlDimension: "数据控权维度",
  AIAgent: "智能体",
  KnowledgeBase: "知识库",
}
