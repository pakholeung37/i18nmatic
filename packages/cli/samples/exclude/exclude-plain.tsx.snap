import { t } from "@terminus/t-console-framework"
const ERROR_MESSAGE = t("请稍后再试")
const review = {
  id: 1,
  name: t("测试"),
}
const reviews = [
  {
    id: 1,
    name: t("张先生"),
  },
  {
    id: 2,
    name: t("李女士"),
  },
]
const typeNameMap: Record<string, string> = {
  Folder: t("文件夹"),
  FolderExpanded: t("文件夹"),
  Model: t("持久模型"),
  ModelView: t("普通模型"),
  Statistic: t("统计模型"),
  Event: t("业务事件"),
  Action: t("扩展服务"),
  ServiceConfig: t("业务配置"),
  Scene: t("页面场景"),
  ServiceDefinition: t("编排服务"),
  Flow: t("业务流"),
  Trigger: t("触发器监听"),
  EventDefinition: t("触发器定义"),
  Process: t("业务流"),
  OverviewProcess: t("业务流程"),
  WorkflowGroup: t("审批流场景"),
  NoticeScene: t("通知场景"),
  PrintScene: t("打印场景"),
  View: t("视图"),
  MenuTree: t("菜单树"),
  TakeCode: t("取号中心"),
  Connector: t("连接器"),
  Menu: t("菜单"),
  Module: t("模块"),
  StateConfig: t("状态配置"),
  Validation: t("校验规则"),
  ImportExportTemplate: t("导入导出场景"),
  Permission: t("功能权限项"),
  DataCondition: t("数据权限规则"),
  DataControlDimension: t("数据控权维度"),
  AIAgent: t("智能体"),
  KnowledgeBase: t("知识库"),
}
